net/nmap/Portfile

```diff
@@ -31,6 +32,12 @@ checksums      rmd160  8418fcef96848a6ae6bbbeee935b34ce047eab5e \
                sha256  d71be189eec43d7e099bac8571509d316c4577ca79491832ac3e1217bc8f92cc \
                size    11102195
 
+# see https://github.com/nmap/nmap/pull/2732
+patchfiles-append 0001-nmap-7.94-fix-32bit.patch
+
+# see https://trac.macports.org/ticket/68543
+patchfiles-append 0002-gcc-4.2-workaround.patch
```

Comment:
I suggest this patch be applied unconditionally, since the patch has no undesirable effect on non-GCC 4.2 platforms, and so that port updaters make sure this patch still works regardless of their platform.